tests/integrations/server/kv_service.rs

```diff
@@ -2185,3 +2185,93 @@ fn test_pessimistic_lock_execution_tracking() {
 
     handle.join().unwrap();
 }
+
+#[test]
+fn test_read_index_execution_tracking() {
```

Comment:
May also need to cover the `batct_get` and `coprocessor` pathes.

Reply:
Added~